book/website/reproducible-research/compendia.md

```diff
@@ -21,21 +21,21 @@ The main purpose is that all elements of a project are published together, so a
 
 ![Cartoon of a research compendium](../../figures/ResearchCompendium.jpg)
 
-## How this will help you/why this is useful
+## How This Will Help You/Why This Is Useful
```

Comment:
This should be updated to 'Motivation' as per the [new template](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/templates/chapter-template/chapter-landing-page.md).